packages/dev-server/README.md

```diff
@@ -112,9 +89,6 @@ By default this middleware will start a development server with Hot Module Repla
 `http://localhost:5000`. To enable HMR with your application, read the documentation of corresponding Neutrino
 preset or middleware.
 
-It is recommended to call this middleware only in development mode when `process.env.NODE_ENV === 'development'`.
```

Comment:
Now that the `@neutrinojs/dev-server` preset only sets the `devServer` option, there's no real real why it shouldn't always be used.